src/components/SideMenuItem/SideMenuItemContainer.js

```diff
@@ -0,0 +1,10 @@
+import { compose, setDisplayName, pure } from 'recompose';
+
+import SideMenuItem from './SideMenuItem';
+
+const enhance = compose(
+  setDisplayName('SideMenuItemContainer'),
+  pure
+);
+
+export default enhance(SideMenuItem);
```

Comment:
Is it really need?

Reply:
Yes, it is needed to prevent extra re-renders when clicking on the same item(without changing active item)

Reply:
Maybe better to use `PureComponent`?

Reply:
Maybe it is better to use `React.memo` then, to keep using functional components?